docs/pages/app.md

```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
 Open the following link to display a _Hello World_ message within an open document. Provide any URL encoded string with the `msg` query parameter.
 
 [`sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World`](sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World)
+
+## Add a Remote Library <a name="addremotelibrary"></a>
+
+You can add a Remote Library to Sketch by providing a url to an appcast file.
+
+### Parameters
+
+Provide an `encoded appcast url` parameter to the `add-library` command.
+
+```sh
+sketch://add-library?url=AppCastURL
+```
+
+#### Example
+
+Open the link below to add a Remote Library to Sketch
```

Comment:
Good example but it's not link, so instead describe _what_ it is:

```suggestion
URL to install the Apple iOS Sketch Library from [Apple Design Resources](https://developer.apple.com/design/resources/).
```

Reply:
Changed.